moskito-webui/src/main/java/net/anotheria/moskito/webui/util/CryptUtil.java

```diff
@@ -0,0 +1,99 @@
+package net.anotheria.moskito.webui.util;
+
+import org.bouncycastle.crypto.InvalidCipherTextException;
+import org.bouncycastle.crypto.engines.BlowfishEngine;
+import org.bouncycastle.crypto.paddings.PaddedBufferedBlockCipher;
+import org.bouncycastle.crypto.params.KeyParameter;
+import org.bouncycastle.util.encoders.Base64;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Utility for decrypting/encrypting using blowfish algorithm
+ * implemented by bouncy castle library.
+ *
+ * Class initializes with key parameter, that be used as a
+ * key for cryptography.
+ *
+ */
+public class CryptUtil {
+
+    /**
+     * Key for decrypting/encrypting
+     */
+    private KeyParameter key;
+
+    public CryptUtil(KeyParameter key) {
+        this.key = key;
+    }
+
+    /**
+     * Decrypts string, encrypted by blowfish algorithm.
+     * Method takes encrypted input in Base64 representation.
+     * Returns string with charset, noted as second method parameter.
+     * @param encryptedString Base64 input for decryption
+     * @param outCharset charset of output string
+     * @return decrypted string
+     * @throws InvalidCipherTextException by bouncy castle library
+     */
+    public String decrypt(String encryptedString, Charset outCharset) throws InvalidCipherTextException {
```

Comment:
I assume this will have concurrency issues, since bouncycastle is not thread safe.